File: Python/Legacy/Stdio.py
#! /usr/bin/env python3
# -*- coding: utf-8 -*-

import sys
from Utility import *

def Stdio():

	prints("""
		こんにちは。私の名前はPython,即ち蛇です。
		あなたのことを教えてください
	""")

	aboutYou = input("あなたは...")

	prints(f"""
		あなたは {aboutYou} ですね。わかりました。

		これから幾つかの出力の仕方を披露しますね。
	""")
	print("後ろに改行を付けます")
	print("後ろに改行を付けません。",end="")
	print("変なものも付けられます",end="☆")
	print("それから...")
	sys.stdout.write("こんな出力も可能です。改行しません。")
	sys.stderr.write("たまにはエラーも出したくなります")
	print("入力してみて (改行してからCtrl+Dで終了)")
	sys.stdin.read()
	print("以上です")

	pnl(2)

runningDirectly(__name__,Stdio)

File: Go/Legacy/Stdio.go
package main

import "fmt"
import "os"
import "log"
import "bufio"

func Stdio(){

	var stdin = bufio.NewScanner(os.Stdin)
	print(clean{},`
		こんにちは。私の名前はGo。
		あまりパッとしませんが,意外にも便利で,低レベルの機能にもアクセスできます
		あなたのことを教えてください。
	`)
	stdin.Scan()
	var aboutYou = stdin.Text()
	print(
		"あなたは ",aboutYou," ですね。わかりました",ln{1},
		"これから基本的な出力の仕方を紹介します",ln{1},
	)

	fmt.Print("後ろに改行を付けません。")
	fmt.Println("後ろに改行を付けます")
	fmt.Fprint(os.Stdout,"こんな出力も可能です。改行しません。") // Fにより出力先を明示できる

	os.Stdout.WriteString("こんな出力も可能です。改行しません。")
	os.Stderr.WriteString("たまにはエラーも出したくなります。")

	print(ln{1})

	log.Print("ログ形式の出力です。改行しません。")

	print(ln{2})

}